installing/installing_aws/installing-aws-customizations.adoc

```diff
@@ -1,14 +1,14 @@
 [id="installing-aws-customizations"]
 = Installing a cluster on AWS with customizations
 include::modules/common-attributes.adoc[]
-:context: install-customizations-cloud
+:context: installing-aws-customizations
```

Comment:
I see two modules where install-customizations-cloud is still used in ifeval statements